examples/gpt-2/README.md

```diff
@@ -1,31 +1,18 @@
 # GPT-2: Pre-trained Langauge Model
 
```

Comment:
Could you update the code examples in the `Other Use Cases` section of README to fit the lastest interfaces?